tests/ci/sync_pr.py

```diff
@@ -53,12 +54,36 @@ def merge_sync_pr(gh, sync_pr):
 
 
 def set_sync_status(gh, pr_info, sync_pr):
-    if not sync_pr or not sync_pr.mergeable:
+    if not sync_pr:
         post_commit_status(
-            get_commit(gh, pr_info.sha), FAILURE, "", "Sync PR failure", "A Sync"
+            get_commit(gh, pr_info.sha), FAILURE, "", "Sync PR not found", "A Sync"
         )
-    else:
+        return
+
+    retries = 0
+    while sync_pr.mergeable_state == "unknown" and retries < 3:
+        retries += 1
+        print(f"Unknown status. Trying to fetch again [{retries}/3]")
```

Comment:
It's a good opportunity to get rid of prints. Let's get used to `logging`

Reply:
print does all that we need here